test/slow/api_spec_slow.js

```diff
@@ -304,6 +304,15 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
             skipPackage: true,
           })).to.eventually.be.rejectedWith(/incompatible with this version/);
         });
+
+        it('can make for the MAS platform successfully', async () => {
```

Comment:
From the build logs, I think some other test might be improperly removing the dir before this test. Possible [here](https://github.com/electron-userland/electron-forge/pull/359/files#diff-3badcddd1644eb71d9e3780b0c254c81L74)